src/styles/themes/default.js

```diff
@@ -43,7 +43,7 @@ const darkTheme = {
     hoverComponentBG: colors.darkHighlightBackground,
     activeComponentBG: colors.darkBorders,
     signInSidebar: colors.green800,
-    sidebar: colors.darkHighlightBackground,
+    sidebar: colors.darkAppBackground,
```

Comment:
Let's make same change for light theme.

Reply:
I don't think there is an easy way to do so (theme switching is WIP). You can just copy the light.js to default.js